test/integration/apiserver/apply/resetFields_test.go

```diff
@@ -89,6 +89,13 @@ func TestApplyResetFields(t *testing.T) {
 					t.Skip()
 				}
 
+				// TODO: fix this test for customresourcedefinitions
```

Comment:
that's kind of an issue...
CRDs should be fine, but their status gets changed before the test finishes, so we get a conflict for .status.conditions
As the test now uses conflicts as a measure if everything is alright, that doesn't work anymore.
Need a better test or another solution to this.

Reply:
Can we find another field that doesn't get updated, to check for conflicts?